tests/eden/eclient/testdata/nw_switch.txt

```diff
@@ -50,7 +50,8 @@ stdout '100% packet loss'
 
 message 'Switching to 2st network'
 eden pod modify pong --networks n2
-test eden.app.test -test.v -timewait 20m RUNNING pong
+test eden.app.test -test.v -timewait 5m PURGING pong
```

Comment:
Don't you have a potential race here if EVE manages to quickly complete the purge operation?
I guess it would involve a reboot of the guest/container hence you should be able to observe that it was in purging state before it is back at running?

Note that in ztests we login to the guest and use the last command to verify that it rebooted to make the test be robust.